bench.mjs

```diff
@@ -1,8 +1,8 @@
 
-import icomesh from './index.js';
+import icomesh from './index.mjs';
```

Comment:
Let's keep the `js` extension. The latest Node ESM spec allows using `.js` with modules as long as we have `type: "module"` in `package.json`.